warp10/src/main/java/io/warp10/script/functions/ECRECOVER.java

```diff
@@ -194,7 +212,6 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
 
           //𝑟−1(𝑠𝑅−𝑧𝐺)  and 𝑟−1(𝑠𝑅′−𝑧𝐺)
```

Comment:
Is that comment still valid? Should it be moved with the `BigInteger rinv` line?